src/main/java/co/rsk/bitcoinj/script/FastBridgeRedeemScriptParser.java

```diff
@@ -25,11 +25,11 @@ public byte[] getDerivationHash() {
         return derivationHash;
     }
 
-    public static Script extractStandardRedeemScript(Script redeemScript) {
-        return ScriptBuilder.createRedeemScript(
+    public static List<ScriptChunk> extractStandardRedeemScript(Script redeemScript) {
+        return (ScriptBuilder.createRedeemScript(
             redeemScript.getNumberOfSignaturesRequiredToSpend(),
             redeemScript.getPubKeys()
-        );
+        )).getChunks();
```

Comment:
ScriptBuilder.createRedeemScript will return a Script object so this doesn't really solve our circular dependency problem. But this is going to change when we complete the parsers refactor, right?

Reply:
You are right, I forgot to update this method regarding the dependency. Is solved now by returning the chunks.sublist(2, chunk.size) instead of creating a script and etc.